ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,187 @@
+    color: E99695
+  - name: C4-mentor
+    description: An easy task where a mentor is available. Please indicate in the issue who the mentor could be.
+    color: "5319e7"
+  - name: C5-reserved
+    description: This issue should not be picked up by anyone unless approved.
+    color: cdadad
+  - name: D0-nice_to_have_audit
+    description: PR contains trivial changes to logic that should be properly reviewed.
+    color: c5def5
+  - name: D1-needs_audit
+    description: PR contains changes to fund-managing logic that should be properly reviewed and externally audited.
+    color: C3EDC0
+  - name: D2-needs_DevRel_audit
+    description: PR contains changes that are relevant for documentation, should be audited by DevRel before merge.
+    color: C0E7CC
+  - name: D2-audited
+    description: PR contains changes to fund-managing logic that has been properly reviewed and externally audited.
+    color: C0E7CC
+  - name: F0-breaks_everything
+    description: This change breaks the underlying networking, sync or related and thus will cause a fork.
+    color: bdadad
+  - name: F1-breaks_authoring
+    description: This change breaks authorities or authoring code.
+    color: e99695
+  - name: F2-breaks_consensus
+    description: This change breaks consensus or consensus code.
+    color: "5319e7"
+  - name: F3-breaks_RPC_API
+    description: This PR changes the RPC API.
+    color: cdadad
+  - name: F4-breaks_runtime_API
+    description: This PR changes the runtime API.
+    color: cdadad
+  - name: F4-breaks_host_function
+    description: This PR changes a host function.
+    color: cdadad
+  - name: I0-panic
+    description: The node panics and exits without proper error handling.
+    color: 17b25d
+  - name: I1-security
+    description: The node fails to follow expected, security-sensitive, behaviour.
+    color: d93f0b
+  - name: I2-bug
+    description: The node fails to follow expected behavior.
+    color: "249899"
+  - name: I3-annoyance
+    description: The node behaves within expectations, however this “expected behaviour” itself is at issue.
+    color: D6075D
+  - name: I4-refactor
+    description: Code needs refactoring.
+    color: 0075ca
+  - name: I5-enhancement
+    description: An additional feature request.
+    color: c2e0c6
+  - name: I6-meta
+    description: A specific issue for grouping tasks or bugs of a specific category.
+    color: a2eeef
+  - name: I7-duplicate
+    description: Issue is a duplicate. Closer should comment with a link to the duplicate.
+    color: D62209
+  - name: I8-wont_fix
+    description: Issue is in principle valid, but this project will not address it. Closer should explain why.
+    color: FB3701
+  - name: I9-invalid
+    description: Issue is invalid. Closer should comment why.
+    color: F76E02
+  - name: I10-optimisation
+    description: An enhancement to provide better overall performance in terms of time-to-completion for a task.
+    color: c5def5
```

Comment:
I'd like to have two labels here: one for CPU optimization, and one for memory usage optimizations.

Reply:
would it be enough if you use `I10-optimisation` with a `T*` label? Then I'd add there `T12-CPU` and `T13-memory_usage`?